client/my-sites/plans/jetpack-plans/product-lightbox/quantity-dropdown.tsx

```diff
@@ -50,8 +50,27 @@ const QuantityDropdown: FC< QuantityDropdownProps > = ( { product, siteId, onCha
 		} );
 	}, [ listPrices.priceTierList, product.productSlug ] );
 
+	const triggerHighlightedIncludes = useCallback( () => {
```

Comment:
From an architectural PoV, I believe this logic doesn't really belong to the quantity dropdown. It applies to highlighted plan details, which don't really have anything to do with the quantity dropdown, from what I understand.

I'd advise moving it to where it belongs (wherever we render these highlightable elements) and potentially abstracting it to be reusable (not a requirement).